io.github.otaxhu.MQTTy.yml

```
@@ -0,0 +1,61 @@
+---
+
+id: io.github.otaxhu.MQTTy
+runtime: org.gnome.Platform
+runtime-version: '48'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+
+command: MQTTy
+
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+
+build-options:
+  append-path: /usr/lib/sdk/rust-stable/bin
+
+modules:
+  - name: blueprint-compiler
+    buildsystem: meson
+    cleanup:
+      - "*"
+    sources:
+      - type: git
+        url: https://gitlab.gnome.org/jwestman/blueprint-compiler
+        tag: v0.16.0
+
+  - name: sass
+    buildsystem: simple
+    cleanup:
+      - "*"
+    build-commands:
+      - cp -r sass/* ${FLATPAK_DEST}/bin
+      - chmod +x ${FLATPAK_DEST}/bin/sass
+    sources:
+      - type: archive
+        url: https://github.com/sass/dart-sass/releases/download/1.86.0/dart-sass-1.86.0-linux-x64.tar.gz
+        sha256: da73e0c544081ed23ce4ec66b1238cbc9a5b80ddfcbde2be41025f3cef40aae6
+        dest: sass
+        only-arches:
+          - x86_64
+      - type: archive
+        url: https://github.com/sass/dart-sass/releases/download/1.86.0/dart-sass-1.86.0-linux-arm64.tar.gz
+        sha256: 547ecca53aab37de4cde1df2341a28fea12c5503bd381ffcc9fd3f61124ed64b
+        dest: sass
+        only-arches:
+          - aarch64
```

```suggestion
      - cp -r * ${FLATPAK_DEST}/bin
      - chmod +x ${FLATPAK_DEST}/bin/sass
    sources:
      - type: archive
        url: https://github.com/sass/dart-sass/releases/download/1.86.0/dart-sass-1.86.0-linux-x64.tar.gz
        sha256: da73e0c544081ed23ce4ec66b1238cbc9a5b80ddfcbde2be41025f3cef40aae6
        only-arches:
          - x86_64
      - type: archive
        url: https://github.com/sass/dart-sass/releases/download/1.86.0/dart-sass-1.86.0-linux-arm64.tar.gz
        sha256: 547ecca53aab37de4cde1df2341a28fea12c5503bd381ffcc9fd3f61124ed64b
        only-arches:
          - aarch64
```

That's the layout. It extract it with `strip: 1`
```
% tar tvf dart-sass-1.86.0-linux-x64.tar.gz 
-rw-r--r-- 0/0           84494 2025-03-17 17:47 dart-sass/src/LICENSE
-rwxr-xr-x 0/0         5804544 2025-03-17 17:47 dart-sass/src/dart
-rw-r--r-- 0/0         5168400 2025-03-17 17:47 dart-sass/src/sass.snapshot
-rwxr-xr-x 0/0             592 2025-03-17 17:47 dart-sass/sass
```